### PR TITLE
Parsing rules fix filter 8

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added filter to parsing rules of the packs:

1. JuniperSRX
2. SonicWallNSv
3. Tenable IO

### CODE REVIEW

File: Packs/JuniperSRX/ReleaseNotes/1_0_9.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Parsing Rules
+##### JuniperSRX Parsing Rule
+- Added a filter in the parsing rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/JuniperSRX/ReleaseNotes/1_0_9.md:3]:
```suggestion
Added a filter in the parsing rule to enhance its logic.
```

File: Packs/SonicWallNSv/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Parsing Rules
+##### SonicwallNSv Parsing Rule
+- Added a filter in the parsing rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/SonicWallNSv/ReleaseNotes/1_0_1.md:3]:
```suggestion
Added a filter in the parsing rule to enhance its logic.
```

File: Packs/Tenable_io/ReleaseNotes/2_1_10.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#### Parsing Rules
+##### Tenable.io Parsing Rule
+- Added a filter in the parsing rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/Tenable_io/ReleaseNotes/2_1_10.md:3]:
```suggestion
Added a filter in the parsing rule to enhance its logic.
```

**Comment by ShirleyDenkberg** [comment]
@yasta5 Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**